CoreAttributeEditorView/src/au/gov/asd/tac/constellation/views/attributeeditor/LabelDropper.java

```diff
@@ -99,9 +102,10 @@
                 }
 
             } catch (final UnsupportedFlavorException | IOException | ClassNotFoundException ex) {
-                Exceptions.printStackTrace(ex);
+                LOGGER.log(Level.SEVERE, ex.getLocalizedMessage(), ex);
             } catch (final ClassCastException ex) {
                 // This exception occurs when dragging a label from Attribute Editor to graph area
+                LOGGER.log(Level.SEVERE, ex.getLocalizedMessage());
```

Comment:
@aldebaran30701 So I omitted the throwable parameter when I was merely adding a log entry (rather than replacing a stack trace print). This was intentional so that nothing was halted that wasn't previously

Reply:
ohh okay so when the exception is passed as a parameter to the logger, it will alter the execution behaviour of the method? 
Most of the other comments may also be invalid as you have noted.